backend/go.mod

```diff
@@ -4,18 +4,24 @@ go 1.15
 
 require (
 	firebase.google.com/go/v4 v4.1.0
+	github.com/ClickHouse/clickhouse-go v1.4.5 // indirect
+	github.com/denisenkom/go-mssqldb v0.10.0 // indirect
 	github.com/buckket/go-blurhash v1.1.0
 	github.com/go-chi/chi v1.5.0
 	github.com/go-chi/cors v1.2.0
+	github.com/go-sql-driver/mysql v1.6.0 // indirect
 	github.com/gofrs/uuid v3.3.0+incompatible // indirect
 	github.com/jackc/fake v0.0.0-20150926172116-812a484cc733 // indirect
 	github.com/jackc/pgx v3.6.2+incompatible
 	github.com/jackc/pgx/v4 v4.10.1
-	github.com/lib/pq v1.9.0 // indirect
+	github.com/lib/pq v1.10.1 // indirect
+	github.com/mattn/go-sqlite3 v1.14.7 // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/pressly/goose v2.7.0+incompatible
 	github.com/shopspring/decimal v1.2.0 // indirect
 	github.com/stretchr/testify v1.6.1
+	github.com/ziutek/mymysql v1.5.4 // indirect
+	golang.org/x/crypto v0.0.0-20210503195802-e9a32991a82e // indirect
```

Comment:
I don't think you added any dependencies in this PR, can you revert this file and go.sum? Correct me if I'm wrong :)